x-pack/plugins/secops/public/pages/styles.tsx

```diff
@@ -16,7 +16,7 @@ export const PageHeader = styled(EuiPageHeader)`
   background-color: ${props => props.theme.eui.euiColorLightestShade};
   position: fixed;
   width: calc(100% - 32px);
-  z-index: 1;
+  z-index: 10;
```

Comment:
I'm starting to wonder if we should consolidate these ala something like https://css-tricks.com/handling-z-index/ or with some of the ideas behind https://github.com/sdgluck/react-z-index. Does Kibana core have a nice strategy for keeping track of all their z-indexes?